index.html

```diff
@@ -1099,5 +1118,34 @@ <h2>Internationalization Considerations</h2>
             guidance in the <a data-cite="vc-data-model/#internationalization-considerations">VC Data Model</a>.
          </p>
       </section>
+      <section class="normative">
+         <h2>IANA Considerations</h2>
+         <section class="normative" id="media-types">
+            <h3>Media Types</h3>
+            <section id="media-types-for-jsonschema">
+               <h2><code>JsonSchema</code></h2>
+               <p>
+                  This specification acknowledges the use of the <code>application/schema+json</code>
+                  and <code>application/json</code> Media Types specifically for identifying a [[JSON-SCHEMA]]
+                  with usage of the <a href="#jsonschema">JsonSchema</a> type, as registered in 
+                  <a href="#json-schema-specifications">JSON Schema specifications</a>.
+               </p>
+               <p>
+                  When using the <a href="#jsonschema">JsonSchema</a> type with a YAML
+                  representation of a JSON Schema, defined by [[OAS]], the types <code>application/openapi+yaml</code>
+                  or <code>application/yaml</code> may be used.
+               </p>
+            </section>  
+            <section class="normative" id="media-types-for-jsonschemacredential">
+               <h2><code>JsonSchemaCredential</code></h2>
+               <p>
+                  This specification acknowledges the use of the <code>application/vc+ld+json</code>,
+                  <code>application/vc+ld+json+jwt</code>, and <code>application/vc+ld+json+sd-jwt</code> Media Types
```

Comment:
`application/vc+ld+json+sd-jwt` is registered in https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-05.html#section-12.2

I'm not sure how to reference that spec though.